sample/ScriptCompositionSample/Program.cs

```diff
@@ -9,26 +10,20 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            var program = new Program();
-            program.Compose();
-
-            program.Greeter.Greet("Hello MEF1!");
-        }
-
-        [Import]
-        public IGreeter Greeter { get; set; }
-
-        public void Compose()
-        {
             // You can add script by script
-            //var catalog = new ScriptCsCatalog(new[] { "Test.csx" }, typeof(IGreeter));
+            //var catalog = new ScriptCsCatalog(new[] { "Test.csx" }, null, typeof(IGreeter));
 
             // Or an entire folder
-            var catalog = new ScriptCsCatalog("Scripts", typeof(IGreeter));
+
+            //Script args can be passed in so the script / script packs can access them.
+            //For example the sample script, the logger script pack is used which depends on these args
+            var scriptArgs = new string[] {"-loglevel", "INFO"};
+
+            var catalog = new ScriptCsCatalog("Scripts", scriptArgs, typeof(IGreeter));
             var container = new CompositionContainer(catalog);
-            var batch = new CompositionBatch();
-            batch.AddPart(this);
-            container.Compose(batch);
+            var greeter = container.GetExportedValue<IGreeter>();
```

Comment:
Sadly I haven't the time to check, but the use of CompositionBatch doesn't work?
Since it is the way I will use it I am a little bit concerned about it.